cluster.yml

```diff
@@ -88,10 +88,13 @@
     - role: dashd-generate-miner
       when: dash_network != "mainnet"
 
-- hosts: faucet-web
```

Comment:
Why changing this back? This was changed to fix the conflicting group + host name. Or does this need a local pull + rebase first?

Reply:
There are multifaucet and insight on these servers now. So I just renamed instances to `web-N` (as others) and brought back old group name.

Reply:
Ok, makes sense if all web apps (e.g. faucet PHP + Insight) on same server.